src/App.js

```diff
@@ -14,11 +15,101 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    goods: [],
+  }
+
+  addGood = (good) => {
+    const { goods } = this.state;
+
+    this.setState({
+      goods: [...goods, good],
+    });
+  }
+
+  removeGood = (good) => {
+    const { goods } = this.state;
+
+    this.setState({
+      goods: goods.filter(elemet => elemet !== good),
+    });
+  }
+
+  deleteGoods = () => {
+    this.setState({
+      goods: [],
+    });
+  }
+
+  isSelected = (goods) => {
+    if (goods.length === 0) {
+      return 'No goods selected';
+    }
+
+    if (goods.length === 1) {
+      return (
+        <>
+          {`${goods} is selected`}
+        </>
+      );
+    }
+
+    return (
+      <>
+        {`${goods.join(', ')} are selected`}
+      </>
+    );
+  }
+
+  render() {
+    const { goods } = this.state;
+
+    return (
+      <div className="app">
+        <h1 className="title">Selected good</h1>
+        <div>
+          {this.isSelected(goods)}
+        </div>
+
+        <button
+          className="button is-light is-danger"
+          type="button"
+          onClick={() => {
+            this.deleteGoods();
+          }}
```

Comment:
якщо метод deleteGoods не приймає параметрі то не потрібно створювати стрілочну функцію
```suggestion
          onClick={this.deleteGoods}
```